openapi_server/__main__.py

```diff
@@ -10,17 +11,18 @@
 
 app = connexion.App(__name__, specification_dir="./openapi/")
 app.app.json_encoder = encoder.JSONEncoder
-# app.app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL", "postgresql:///test-cfd")
+logging.info("Looking for DATABASE_URL, if not found an in-memory db will be used")
 app.app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv(
-    "DATABASE_URI", "sqlite:///:memory:"
-)  # can this work if there's no db to connect to?
+    "DATABASE_URL", "sqlite:///:memory:"
```

Comment:
@gmemstr An in memory sqllite database is used if the database URL is not present